source/clear-linux/guides/maintenance/autospec.rst

```diff
@@ -23,8 +23,10 @@ This guide requires that you:
 
 .. _install-tooling:
 
-Install the |CL| tooling framework
-==================================
+Install the |CL| development tooling framework
```

Comment:
Simplify section title to remove "Clear Linux OS" e.g. "Install development tooling framework"

Reply:
I added this to be more specific because the tooling is created by CL and specific to CL.